Android/app/build.gradle

```diff
@@ -15,6 +15,31 @@ try {
     keystoreProperties['storePassword'] = ''
 }
 
+// Go backend build constants
+def goSourceDir = "${projectDir}/src/go"
+def goSourcePackages = ["${goSourceDir}/intra",
```

Comment:
Why do we need a `/intra` subdirectory? I'd say Intra is implicit here, since we are in the Intra folder already. This seems redundant.

Reply:
Because we need those classes to be put under the `intra` package, for example these two classes: https://github.com/Jigsaw-Code/Intra/blob/master/Android/app/src/main/java/app/intra/net/go/GoIntraListener.java#L29-L30

Reply:
Ah, ok. BTW, you can use `-javapkg` to specify the package (https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile#hdr-Build_a_library_for_Android_and_iOS), but that doesn't need to be in this PR.